### PR TITLE
fix: add aiohttp to deps

### PR DESCRIPTION
**Pull request by ayal**
## Description

add aiohttp to deps after this change: https://github.com/TomerFi/aioswitcher/commit/7c45f8bc3ed48e42df56f888a0ef09d4d6624a8a

## Checklist

- [x] I have followed this repository's contributing guidelines.
- [x] I will adhere to the project's code of conduct.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -23,6 +23,7 @@ exclude = [ ]
   [tool.poetry.dependencies]
   python = "^3.9.0"
   pycryptodome = ">=3.18.0"
+  aiohttp = "^3.10.3"
```
Comment (thecode) [inline pyproject.toml]:
```suggestion
  aiohttp = ">3.10.3"
```
Can we just make sure it is above the minimum version?

**Review by thecode — DISMISSED** [review]

**Comment by TomerFi** [comment]
@ayal, Before merging, can you please document what's wrong with version 4.0.0 of _aiohttp_ in the PR body?

**Comment by thecode** [comment]
> @ayal, Before merging, can you please document what's wrong with version 4.0.0 of _aiohttp_ in the PR body?

I just noticed this was added in a later [commit](https://github.com/TomerFi/aioswitcher/pull/801/commits/0b4cdca4b7958a5a0868413524e6563e7a389204) after I approved the PR.
The main idea was to make sure we don't need to create a new release when `aiohttp` will reach version 4. 

As the one of the usages for `aiohttp` is going to be in Home Assistant and the most active developer in `aiohttp` is one of the Home Assistant core members I don't think there is a need to restrict anything. If there will be something breaking in `aiohttp` it will not be bumped in Home Assistant until all dependencies are compatible, if we restrict here we just force that we will not be compatible, but we don't know now anything that is not compatible.

**Comment by ayal** [comment]
@thecode  @TomerFi
Changed it now to allow any version.
I'm used to a different version-pinning practice,
i.e pin to a specific version and then use some bot (for example renovate) for upgrading versions (by a pr to test the upgrade, then auto-merge if successful). Major versions are specially indicative to breaking changes so should be even more careful.

**Review by thecode — APPROVED** [review]

**Review by TomerFi — APPROVED** [review]

**Pull request merged**